com.github.fwbuilder.appdata.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.github.fwbuilder</id>
+  <provides><id>com.github.fwbuilder.desktop</id></provides>
```

Comment:
```suggestion
```
You're only providing `com.github.fwbuilder which is the current id of the component, there's no need to re-add it. provides id is only useful if you migrate from an old id.